addons/hr_expense/tests/test_expenses.py

```diff
@@ -211,17 +211,14 @@ def test_expense_main_flow(self):
             (self.analytic_account_1 | self.analytic_account_2).unlink()
 
         # Unlinking moves
```

Comment:
Put the old behavior https://github.com/odoo/odoo/commit/01b87f1230beac0568f4e3b1b76e547909506892#diff-799ab636d9774ba187a04433d7f836b860ce15a20c5e6f7dc414eebdb2bc670c

Reply:
THX, there are other changes to revert in expense tests but I'll take care of it